pkg/virt-handler/vm.go

```diff
@@ -334,9 +340,23 @@ func (d *VirtualMachineController) hasTargetDetectedDomain(vmi *v1.VirtualMachin
 
 func (d *VirtualMachineController) clearPodNetworkPhase1(vmi *v1.VirtualMachineInstance) {
```

Comment:
@vatsalparekh looks like you don't need VMI object but only its UID, you can change the function arguments